components/accept-financial-contributions/StripeOrBankAccountPicker.js

```diff
@@ -30,10 +37,24 @@ const Image = styled.img`
   }
 `;
 
+const ConnectedAccountCard = styled(Flex)`
+  box-shadow: 0 1px 3px 2px rgba(46, 77, 97, 0.1);
+  border-radius: 16px;
+`;
+
+const GreenCheckbox = styled(CheckboxChecked)`
+  color: ${themeGet('colors.green.700')};
+`;
+
 class StripeOrBankAccountPicker extends React.Component {
   static propTypes = {
-    collective: PropTypes.object.isRequired,
+    data: PropTypes.object.isRequired,
+    intl: PropTypes.object.isRequired,
     router: PropTypes.object,
+    hostCollectiveSlug: PropTypes.string.isRequired,
+    LoggedInUser: PropTypes.object,
```

Comment:
I find `this.props.LoggedInUser` confusing because it's not coming as usual from Apollo but from the parent component. After that, the usage where we're looking either in `LoggedInUser` and `hostOrganization` is a bit disturbing to me.

If I understand your intent well, I think the best would be to just have a `props` named `host` and use it like that:

```
            <StripeOrBankAccountPicker
              collective={collective}
              host={LoggedInUser.collective}
              addHost={this.addHost}
            />
```

Not a blocker, I can propose a PR later.

Reply:
PR: https://github.com/opencollective/opencollective-frontend/pull/4043

Reply:
I have commented on your PR - unless there's anything else blocking at this point, let me know if I can merge this one, then I can put up my next PR for the organization flow and you can see why I did things this way.